Dockerfile.apt-cacher

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:focal@sha256:cf31af331f38d1d7158470e095b132acd126a7180a54f263d386da88eb681d93
+
+VOLUME ["/var/cache/apt-cacher-ng"]
+RUN    apt-get update && apt-get install -y apt-cacher-ng
+
+EXPOSE 3142
+CMD    chmod 777 /var/cache/apt-cacher-ng && /etc/init.d/apt-cacher-ng start && tail -f /var/log/apt-cacher-ng/*
+
+RUN cat /etc/hosts
```

Comment:
Strange that nothing "off the shelf" already existed? Seems like a useful feature

Reply:
yes, no official image, don't want to use unknown image, we can move this to a new repo if you like the general approach.

We can also install and run apt-cacher on ubuntu worker, so no need for an docker image

Reply:
now using native ubuntu package